table-to-blt.mjs

```diff
@@ -157,14 +160,21 @@ function skips(ballot) {
   return ballot.some((v,i) => (v.rank - i) >= 2)
 }
 
-const file = process.argv[2];
-const nbseats = process.argv[3];
-const electionname = process.argv[4] === undefined ? ("Election " + new Date().toISOString().slice(0, 10)) : process.argv[4];
-const sortballots = process.argv[5] === undefined ? "true" : process.argv[5];
-const shownames = process.argv[6] === undefined ? "false" : process.argv[6];
-
-main(file, nbseats, electionname, sortballots, shownames)
-  .catch(err => {
-    console.log(`Something went wrong: ${err.message}`);
-    throw err;
-  });
+// Detect if we're running from the node CLI
+if (import.meta.url === ('file:///'+ process?.argv[1].replace(/\\/g,'/')).replace(/\/{3,}/,'///')) {
```

Comment:
This allows importing the script without the main function running automatically (which is needed for testing).
The syntax to detect if we're running in CLI mode is horrendous (there are ongoing discussions to improve the situation)